src/components/App/App.tsx

```diff
@@ -5,12 +5,35 @@ import {
     Switch, Route, useRouteMatch
 } from "react-router-dom"
 
+// import User from '../UserProfile/UserProfile'
+
 import Users from '../UserTile/usertile';
 import Header from '../LoginHeader/loginHeader';
 import UserSearchBox from '../UserSearch/usersearch';
 import UserProfile from '../UserProfile/UserProfile'
 
-const App = () => {
+export interface MatchParams{
+    id?: string;
+  }  
+
+  export interface User {
+    id?: number;
+    first_name: string;
+    last_name: string;
+    email?: string;
+    bgImg?: string;
+    profile_photo?: string;
+    title?: string;
+    department?: string;
+    client?: string;
+    city?: string;
+    state?: string;
+    phone?: string;
+    quote?: string;
+    quoteAuthor?: string;
+}
+
+const App: React.FC<MatchParams> = () => {
 
     const loggedInUser = {
```

Comment:
If your app fully intends on using the interface User with getters and setters, then sample data should also be mocked this way so you don't have to change what is passed to different react components. Helper methods may also be useful, such as `GetUser(..)`, that `loggedInUser` can be assigned to so that your mock implementation of `GetUser` can return any data you want, random, static, or from a test endpoint. Typescript for the win.

(notice that interfaces should start with "I")
const loggedInUser: IUser = {
    id: 71202137821,
    first_name: "Rohith",
    last_name: "Palagiri",
    quoteAuthor: 'Thomas Edison'
};